pkg/client/client.go

```diff
@@ -82,8 +83,26 @@ type CacheOptions struct {
 	// Unstructured is a flag that indicates whether the cache-backed client should
 	// read unstructured objects or lists from the cache.
 	Unstructured bool
+	// MissPolicy determines how the client should behave when the underlying cache is
+	// configured to fail on a miss. See the CacheMissPolicies for documentation for each
+	// policy.
+	MissPolicy CacheMissPolicy
```

Comment:
Hm, does this only apply when the underlying cache is configured in fail mode?

Reply:
Yes, the GoDoc mentions this. There is no possibility of a cache miss in the backfill case.